lib/ansible/modules/systemd.py

```diff
@@ -493,7 +493,11 @@ def main():
 
             # check systemctl result or if it is a init script
             if rc == 0:
-                enabled = True
+                if out.strip() in ('enabled-runtime',):
+                    # enable-runtime is 'enable for only this boot of the system'
+                    enabled = False
+                else:
+                    enabled = True
```

Comment:
Let's keep it simple.
```suggestion
                # enable-runtime is 'enable for only this boot of the system'
                enabled = out.strip() != 'enabled-runtime'
```

Reply:
we might want to add a warning as user might not be aware of this 'enabled but not enabled' state

'found enabled-runtime but this does not mean it is really enabled'

Reply:
link to systemd docs might be nice too

Reply:
Yeah, I totally agree with this.